components/brave_shields/browser/ad_block_service.cc

```diff
@@ -66,27 +66,36 @@ void AdBlockService::ShouldStartRequest(
     const GURL& url,
     blink::mojom::ResourceType resource_type,
     const std::string& tab_host,
+    bool aggressive_blocking,
```

Comment:
@bridiver we have to refactor this hierarchy so that the facade method doesn't share its signature with general adblock / regional ablock / etc.

Here we add a parameter to the facade method, and that parameter is useless for all other methods

Reply:
I agree a lot of refactoring is needed in general. Several PRs are in the works